packages/opam-depext/opam-depext.1.1.4/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: [
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+  "Anil Madhavapeddy <anil@recoil.org>"
+]
+authors: [
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+  "Anil Madhavapeddy <anil@recoil.org>"
+]
+license: "LGPL-2.1 with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocaml/opam-depext"
+bug-reports: "https://github.com/ocaml/opam-depext/issues"
+depends: ["ocaml"]
+available: opam-version >= "2.0.0~beta5"
```

Comment:
Should this be marked incompatible with opam 2.1~alpha? (not sure wish version brought back compatibility with it)

Reply:
Only the 2.1.0~alpha, it was fixed in [alpha2](https://github.com/ocaml/opam/releases/tag/2.1.0-alpha2)